src/components/accordion/index.md.njk

```diff
@@ -17,83 +17,119 @@ The accordion component lets users show and hide sections of related content on
+If you decide to use one of these components, consider if:
+- the user needs to look at more than one section at a time — an accordion can show multiple sections at a time, unlike tabs
+- the user needs to switch quickly between sections — tabs can show content without pushing other sections down the page, unlike accordions
+- there are many sections of content — accordions can fit more sections as they’re arranged vertically, unlike tabs which are arranged horizontally
+- there’s only one or two pieces of short, less important content — the details component is more suitable as it’s visually smaller and less prominent than an accordion or tabs
 
 ## How it works
 
 There are 2 ways to use the accordion component. You can use HTML or, if you’re using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.
 
 The accordion component uses JavaScript. When JavaScript is not available, users will see all the content displayed with the section labels as headings.
 
-### Find out how your users interact with the content
+### Section heading buttons
+
+An accordion will usually start with all sections hidden. To show a section, the user can interact anywhere in the heading button.
 
-Users might need the sections they have opened to stay open if they leave and then return to the page. You can configure the accordion component to stay open.
+The heading button includes all of these areas:
+- heading text
+- summary text (if you decide to add one)
+- call-to-action text to 'show' or 'hide'
+
+For users of screen readers, all the text in the button will be read as a single statement (separated by commas to allow for slight pauses). There’s also some visually hidden content in the heading text to help announce the call-to-action as 'show this section' or 'hide this section'.
+
+{{ example({group: "components", item: "accordion", example: "default", html: true, nunjucks: true, open: false, size: "xl"}) }}
 
-Do user research to find out the best order for the sections.
+#### Write clear button text
 
-### Use clear labels
+Write the heading and summary text like any other button text. Use sentence case, describe the content that will show, and keep it short."
+Users of screen readers might find it difficult to navigate the accordion if the button text is too long.
 
-Accordions hide content, so the labels need to be clear. If necessary, you can add a summary line to help users understand what is in the section.
+If you struggle to come up with clear button text, it might be because the way you’ve separated the content is not clear. Organise sections in a way that makes sense to users, based on their needs.
+
+#### Adding a summary line
+
+Only add a summary line if it’s actually needed, as it's likely to make the button text too long.
+
+If you’ve decided that you need the summary line, you must make it as short as possible.
```

Comment:
Should we try to revise the content in the below example to use shorter summary lines? Just realised that it doesn't actually follow the spirit of our guidance.

Reply:
Probably something to look at later after we publish, perhaps with the service manual team on their live pages?
We've haven't seen anything live that fits and I'd find it difficult to make a thought-out example.

I'll suggest it in the accordion retro so we can do it as a next step.

Reply:
Added as issue to look at after release:
https://github.com/alphagov/govuk-design-system/issues/1999